src/internal/connector/graph/metadata/metadata.go

```diff
@@ -0,0 +1,16 @@
+package metadata
```

Comment:
not sure what this package should be called or where this code should live exactly. We can't put it in the `graph` package because that creates an import cycle. Happy to take suggestions on a better name/location

Reply:
Maybe in `internal/connector`, but this seems OK too.

Reply:
I think `internal/connector` is not possible because it will create an import cycle between `onedrive` and `connector` :/